pppd/pppd.8

```diff
@@ -637,6 +637,9 @@ Set the maximum number of LCP terminate-request transmissions to
 Set the LCP restart interval (retransmission timeout) to \fIn\fR
 seconds (default 3).
 .TP
+.B lcp\-rtt\-file \fIfilename
+Sets the file where the RTT of LCP echo-request frames will be logged.
```

Comment:
Since this is documentation, please change "the RTT" to "the round-trip time (RTT)".